From 1e3de9d7b5f0cfb6a5fb743a9c2f9d19f51a3d05 Mon Sep 17 00:00:00 2001
From: OpenAI Assistant <assistant@example.com>
Date: Tue, 25 Feb 2025 00:00:00 +0000
Subject: [PATCH] imx8mp_evk: set SPL FIT load address

Set CONFIG_SPL_LOAD_FIT_ADDRESS so that SPL loads the FIT image from
0x40200000.

---
 configs/imx8mp_evk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/imx8mp_evk_defconfig b/configs/imx8mp_evk_defconfig
index 4e7f1cd51b..e28a1a37c8 100644
--- a/configs/imx8mp_evk_defconfig
+++ b/configs/imx8mp_evk_defconfig
@@
 CONFIG_FIT=y
 CONFIG_FIT_EXTERNAL_OFFSET=0x3000
 CONFIG_SPL_LOAD_FIT=y
+CONFIG_SPL_LOAD_FIT_ADDRESS=0x40200000
 CONFIG_SYS_BOOTM_LEN=0x2000000
-- 
2.34.1

